tfhe/docs/fundamentals/compress.md

```diff
@@ -41,6 +41,84 @@ fn main() {
 }
 ```
 
+## Ciphertexts compressed while encrypted
+
+Ciphertexts can also be compressed while encrypted.
+They can be encrypted after any number of homomorphic operations.
+
+The API uses a list which can compress many ciphertexts of different types.
+Using a single list allows a better compression ratio.
```

Comment:
A list to compress many ciphertexts of different types is provided.
Using a single list to compress several ciphertexts allows to achieve a better compression ratio.

?

Reply:
Done